File: BasicMaths/CountDigitsInN/CountDigits.py
import math
class CountDigits:
    def digitCount(self,number):
        count = int(math.log10(number) + 1)

        print(count)
    
    def run(self):
        n = int(input("Enter the number: "))
        self.digitCount(n)

countDigits = CountDigits()
countDigits.run()

File: BasicMaths/CountDigitsInN/CountDigits.java
package BasicMaths.CountDigitsInN;

import java.util.Scanner;

public class CountDigits {
    private static void countDigits(int number) {
        int count = (int) (Math.log10(number) + 1);
        System.out.println(count);
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        countDigits(n);
        in.close();
    }
}
